projects/manager/src/app-theme.scss

```diff
@@ -0,0 +1,83 @@
+@use '@angular/material' as mat;
+
+// Material Base Styles
+@include mat.all-component-typographies();
+@include mat.core();
+
+// Define Colors
+$geoengine-main-green: (
+    50: #e7f5e7,
+    100: #c6e5c3,
+    200: #a1d49d,
+    300: #7ac476,
+    400: #5db858,
+    // main color
+    500: #3fab39,
+    600: #359c31,
+    700: #298a26,
+    800: #1c7a1b,
+    900: #005b04,
+    contrast: (
+        50: #000000,
+        100: #000000,
+        200: #000000,
+        300: #000000,
+        400: #000000,
+        500: #000000,
+        600: #ffffff,
+        700: #ffffff,
+        800: #ffffff,
+        900: #ffffff,
+    ),
+);
+
+$geoengine-mid-blue: (
+    50: #e1f5fa,
+    100: #b3e4f3,
+    200: #83d3eb,
+    300: #56c1e4,
+    400: #36b4e0,
+    500: #18a7dd,
+    600: #0e99cf,
+    // main color
+    700: #0087bc,
+    800: #0176a9,
+    900: #005687,
+    contrast: (
+        50: #000000,
+        100: #000000,
+        200: #000000,
+        300: #000000,
+        400: #000000,
+        500: #000000,
+        600: #000000,
+        700: #000000,
+        800: #ffffff,
+        900: #ffffff,
+    ),
+);
+
```

Comment:
1. doesn't it make sense for every app to have an individual theme?
2. you said you didn't want the manager to depend on the core, because it is very specific to map applications. We could also think of a core-core where everything depends on and where we would have stuff common to all apps...

Reply:
1. I think in general we should strive for one look and feel.
2. The style isn't defined in the core but for every individual app. Maybe we can just add one style-file somewhere and import it. Or… as you suggest, add one plain JS module that provides one file.

Reply:
where would you define it?

Reply:
Try the projects folder? or some subdir in it? And if this does not work, we will leave it as is for now.

Reply:
and how would I include it?

Reply:
with  @use or @include I guess.

Reply:
I tried
`@use '../../app-theme.scss';`
but then I get "[ERROR] There is no module with the namespace "mat"."

@include doesn't work because it expects an identifier.